practice-app/practice/cron.py

```diff
@@ -0,0 +1,57 @@
+import sys
+import requests
+from api.models import Equipment, Parity
+from django_cron import CronJobBase, Schedule
+from django.utils.timezone import make_aware
+from datetime import datetime
+
+
+class GetExchangeRates(CronJobBase):
+    RUN_EVERY_MINS = 5
+
+    schedule = Schedule(run_every_mins=RUN_EVERY_MINS)
+    code = 'practice.GetExchangeRates'
```

Comment:
where exactly are we using this variable? (sorry if i'm missing)

Reply:
I guess it is necessary for django-cron app to work. Checkout [its doc. ](https://django-cron.readthedocs.io/en/latest/installation.html)